aws_lambda_powertools/utilities/parser/models/apigw.py

```diff
@@ -83,10 +90,3 @@ class APIGatewayProxyEventModel(BaseModel):
     stageVariables: Optional[Dict[str, str]]
     isBase64Encoded: bool
     body: str
-
-    @root_validator()
-    def check_message_id(cls, values):
-        message_id, event_type = values.get("messageId"), values.get("eventType")
```

Comment:
@risenberg-cyberark  - tried to get this to be triggered, but `values` was for the parent and so the `messageId` was actually inside of the `requestContext`, but referencing that here was causing other issues.

Another solution would be just to remove this check, if AWS is doing this for us?

Reply:
you can remove it, it's not that critical